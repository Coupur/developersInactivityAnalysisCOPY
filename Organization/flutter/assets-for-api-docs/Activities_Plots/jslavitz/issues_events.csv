id,date,event,creator_login
2166046789,2019-02-26 20:20:51,review_requested,jslavitz
2166047124,2019-02-26 20:20:58,review_requested,jslavitz
2166050985,2019-02-26 20:22:33,renamed,jslavitz
2172240693,2019-02-28 20:36:04,review_dismissed,jslavitz
2172240695,2019-02-28 20:36:04,review_dismissed,jslavitz
2089948365,2019-01-22 20:57:13,referenced,jslavitz
2089948366,2019-01-22 20:57:13,merged,jslavitz
2089948379,2019-01-22 20:57:14,closed,jslavitz
