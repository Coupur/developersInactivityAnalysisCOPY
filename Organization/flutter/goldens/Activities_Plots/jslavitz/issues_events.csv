id,date,event,creator_login
2159049941,2019-02-22 21:51:53,head_ref_force_pushed,jslavitz
2159057146,2019-02-22 21:55:41,head_ref_force_pushed,jslavitz
2182908595,2019-03-06 02:47:13,head_ref_force_pushed,jslavitz
2182912135,2019-03-06 02:49:51,head_ref_force_pushed,jslavitz
2102436396,2019-01-29 01:51:01,referenced,jslavitz
2102436399,2019-01-29 01:51:01,merged,jslavitz
2102436401,2019-01-29 01:51:01,closed,jslavitz
2090360875,2019-01-23 00:45:45,referenced,jslavitz
2090360876,2019-01-23 00:45:45,merged,jslavitz
2090360881,2019-01-23 00:45:46,closed,jslavitz
2084241522,2019-01-19 00:42:04,head_ref_force_pushed,jslavitz
2084243579,2019-01-19 00:44:10,head_ref_force_pushed,jslavitz
2090275673,2019-01-22 23:43:04,referenced,jslavitz
2090275675,2019-01-22 23:43:04,merged,jslavitz
2090275680,2019-01-22 23:43:04,closed,jslavitz
